crates/sui-data-ingestion-core/src/reader.rs

```diff
@@ -220,19 +220,22 @@ impl CheckpointReader {
         })
         .await?;
 
+        let mut read_source: &str = "local files";
```

Comment:
looks good. Just nit: maybe shorter log? (i.e. `local` and `remote`)?

Reply:
every bit counts ;)